### PR TITLE
Adding v0.9 API version for device management and certificate management

### PR DESCRIPTION
**Pull request by lasanthaDLPDS**
ex :- <host>:<port>/api/device-mgt/v0.9/devices/0.9.0/type/{device-type}/id/{device-id}

### CODE REVIEW

**Review by madawas — APPROVED** [review]

**Review by Kamidu — COMMENTED** [review]
Update the Copyright text accordingly.

File: components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/CertificateList.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    public void setList(List<CertificateResponse> certificates) {
+        this.certificates = certificates;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("{\n");
+
```
Comment (Kamidu) [inline components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/CertificateList.java:46]: Unwanted Empty line here

File: components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/ErrorListItem.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("errorItem {\n");
+
```
Comment (Kamidu) [inline components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/ErrorListItem.java:72]: Unwanted Empty Line

File: components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/CertificateList.java (hunk 1 above)
Comment (lasanthaDLPDS) [reply components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/CertificateList.java:46]: This is fixed e30ce5bcbda6caa6001e8af139d845116f18e141

File: components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/ErrorListItem.java (hunk 2 above)
Comment (lasanthaDLPDS) [reply components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.v09.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/beans/ErrorListItem.java:72]: This is fixed e30ce5bcbda6caa6001e8af139d845116f18e141

**Comment by lasanthaDLPDS** [comment]
Updated the copyright texts. 1d53023fdc66731eb25b7596726a500207339e86 831fdcb3357a7230b92cd7674fe98cb7d3b55aea 0aef14bdbe19e89e871835d75430f3692595bcb5

**Pull request merged**